AK/Math.h

```diff
@@ -36,6 +36,12 @@ template<size_t value>
 constexpr size_t product_odd() { return value * product_odd<value - 2>(); }
 }
 
+#ifdef __clang__
+#    define CONSTEXPR ALWAYS_INLINE
```

Comment:
I don’t think naming this CONSTEXPR when it won’t be in some compilers isa good idea, please invent a new name.

Reply:
Well I am even thinking of just marking these as just "inline"/"static inline",
because some of the implementations are gonna be big enough, to be not ideal to inline in the not constant evaluated case...
Sadly we cant mutate constexpr-ness in specializations, can we do so for __attribute__'s though?

Reply:
also I am bad at naming...

Reply:
Please drop this change. There is no real benefit to forcing these to be inlined, and just makes the code more confusing. Even if these never produce constant expressions on Clang, `constexpr` implies `inline` regardless.